### ISSUE TITLE
Jasmine 'expect(…).withContext is not a function error when I switched to "@wdio/jasmine-framework" V : 8.3.3

### ISSUE THREAD

**Issue by aswinchembath** [jasmine/jasmine#1997]
### Your question

Getting Jasmine 'expect(…).withContext is not a function'error when I switched to "@wdio/jasmine-framework" V : 8.3.3function when am working with webdriver io 8.3

### Example code that demonstrates the problem

```JavaScript
expect(…).withContext('Failure msg').toBe(true)
```

### jasmine-core version

4.5.0

### Versions of other relevant packages

@wdio/jasmine-framework : 8.3.3

### Node.js or browser version

Node 19.3

### Operating System

Windows 10

**Comment by sgravrock** [comment]
It looks like `@wdio/jasmine-framework` monkey patches much of Jasmine's public interface. Of particular relevance, it replaces Jasmine's `expect` with one from `expect-webdriverio`:

https://github.com/webdriverio/webdriverio/blob/cc6deb72e7c4f064074ca8a2e101da7c08db8b4b/packages/wdio-jasmine-framework/src/index.ts#L198-L201

The `expect` provided by `expect-webdriverio` [is quite different from Jasmine's expect API](https://webdriver.io/docs/api/expect-webdriverio). 

Basically you aren't actually using Jasmine, you're using a webdriverio framework that uses some pieces of Jasmine. If you're unhappy with the way it works you can certainly log bugs with `@wdio/jasmine-framework`, although in this case I expect they'd say that it's working as intended.
